consensus/src/config.rs

```diff
@@ -30,5 +30,8 @@ pub const DEFAULT_BLOCK_GAS_LIMIT: u64 = 5 * 1_000_000_000;
 
 pub const RELAX_ITERATION_THRESHOLD: u8 = 10;
 
+/// Emergency mode is enabled only for the last N iterations
+pub const EMERGENCY_MODE_ITERATION_THRESHOLD: u8 = CONSENSUS_MAX_ITER - 5;
```

Comment:
2 questions:

1) Can we lower this to increase the number of emergency iterations? 4  iteration aren't enough imo @fed-franz 
2) If we vote for former iterations during the EMERGENCY_MODE, what is the RELAX_ITERATION_THRESHOLD needed for? Aren't they the same thing?

Reply:
1. Yes, we can. What do you think would be more appropriate? 10 iterations?
2. I think `RELAX_ITERATION_THRESHOLD` was initially meant to allow blocks not including failed iterations, but it's probably obsolete now. I would remove it

Reply:
Any decision on that? Shall we increase the default `EMERGENCY_MODE_ITERATION_THRESHOLD`?

Should we replace RELAX_ITERATION_THRESHOLD with EMERGENCY_MODE_ITERATION_THRESHOLD const?

@fed-franz @herr-seppia

Reply:
I think 10 iterations is enough for the emergency mode, and any more would start being problematic (since each iteration is "concurrent" to the other emergency ones and potentially generating forks).

As for the RELAX_ITERATION_THRESHOLD I'm not totally sure it should be replaced.
My doubt is: would it be prohibitive to deal with MAX_ITERATIONS-EMERGENCY_MODE_ITERATION_THRESHOLD (let's say 245) NilQuorum certificates in a block? 
If not, than I would remove RELAX_ITERATION_THRESHOLD. Otherwise, we probably need it and set it to an appropriate number

Reply:
> My doubt is: would it be prohibitive to deal with MAX_ITERATIONS-EMERGENCY_MODE_ITERATION_THRESHOLD (let's say 245) NilQuorum certificates in a block?

We should benchmark it. 
For now, I'll stick with `EMERGENCY_MODE_ITERATION_THRESHOLD=9` and keep the RELAX_ITERATION_THRESHOLD as it is. When we start testing, the scenario can be reproduced and the impact measured.

Reply:
Done